### PR TITLE
trigger predictions before sending events

### PR DESCRIPTION
**Pull request by lgenuit**
Add code to trigger model predictions before sending the events.

### CODE REVIEW

File: content_root/tutorial/06 publish_event.ipynb [hunk 1]
```diff
@@ -80,6 +80,24 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# dataset_id = 'winequality'\n",
+    "# client.trigger_model_predictions(project_id, model_id, dataset_id)"
```
Comment (osmansarood) [inline content_root/tutorial/06 publish_event.ipynb:98]: Not sure, why this is commented out?

**Review by osmansarood — APPROVED** [review]

File: content_root/tutorial/06 publish_event.ipynb (hunk 1 above)
Comment (lgenuit) [reply content_root/tutorial/06 publish_event.ipynb:98]: Just if the model is not a new one and they already have predictions, they don't need this step (I added some text to describe why).

Comment (osmansarood) [reply content_root/tutorial/06 publish_event.ipynb:98]: sounds good.

**Comment by osmansarood** [comment]
lgtm. Thanks Lea!

**Comment by osmansarood** [comment]
In case you haven't done it already, it might be best to test this out on a new model.

**Pull request merged**